frontend/src/server-middleware/robots.js

```diff
@@ -1,5 +1,45 @@
 const { LOCAL, PRODUCTION } = require("../constants/deploy-env")
 
+const AI_ROBOTS_CONTENT = `
```

Comment:
Would it be easier to maintain  a list of user-agent names, and create the block string using it:
```
uaList.map(ua => `User-agent: ${ua}\nDisallow: /\n`).join("\n")
```